modules/rosa-sts-creating-a-cluster-quickly-ocm.adoc

```diff
@@ -0,0 +1,276 @@
+====
+.. Optional: Select *Encrypt persistent volumes with customer keys* if you want to provide your own AWS Key Management Service (KMS) key Amazon Resource Name (ARN). The key is used for encryption of persistent volumes in your cluster.
+.. Click *Next*.
+
+. On the *Default machine pool* page, select a *Compute node instance type*.
++
+[NOTE]
+====
+After your cluster is created, you can change the number of compute nodes in your cluster, but you cannot change the compute node instance type in the default machine pool. The number and types of nodes available to you depend on your {product-title} subscription.
+====
+
+. Optional: Configure autoscaling for the default machine pool:
+.. Select *Enable autoscaling* to automatically scale the number of machines in your default machine pool to meet the deployment needs.
+.. Set the minimum and maximum node count limits for autoscaling. The cluster autoscaler does not reduce or increase the default machine pool node count beyond the limits that you specify.
+** If you deployed your cluster using a single availability zone, set the *Minimum node count* and *Maximum node count*. This defines the minimum and maximum compute node limits in the availability zone.
+** If you deployed your cluster using multiple availability zones, set the *Minimum nodes per zone* and *Maximum nodes per zone*. This defines the minimum and maximum compute node limits per zone.
++
+[NOTE]
+====
+Alternatively, you can set your autoscaling preferences for the default machine pool after the machine pool is created.
+====
+
+. If you did not enable autoscaling, select a compute node count for your default machine pool:
+* If you deployed your cluster using a single availability zone, select a *Worker node count* from the drop-down menu. This defines the number of compute nodes to provision to the machine pool for the zone.
+* If you deployed your cluster using multiple availability zones, select a *Worker node count (per zone)* from the drop-down menu. This defines the number of compute nodes to provision to the machine pool per zone.
+
+. Optional: Expand *Edit node labels* to add labels to your nodes. Click *Add label* to add more node labels and select *Next*.
+
+. In the *Cluster privacy* section, select *Public* or *Private* to use either public or private API endpoints and application routes for your cluster.
++
+[IMPORTANT]
+====
+If you are using private API endpoints, you cannot access your cluster until you update the network settings in your cloud provider account.
+====
+
+. To install the cluster in an existing AWS Virtual Private Cloud (VPC):
+* Optional: If you opted to use public API endpoints, you can select *Install into an existing VPC* to install your cluster into an existing VPC.
```

Comment:
Is this a one-or-the-other situation? Like they would either use public or private API endpoints? Just wondering why this one is labeled optional

Reply:
The selection of public or private endpoints in the preceding step is either/or. If you opt to use a public endpoint, you can optionally install into an existing VPC. If you opt to use a public endpoint, you must use an existing VPC, which is why the subsequent bullet is not labelled optional. I will try to make this clearer in the procedure somehow.

Reply:
I have updated this step to relate specifically to the public endpoint scenario. Then, I've added a note underneath to clarify that you must use an existing VPC if you opted to use a private endpoint. I think this is less ambiguous now about what is optional:

```
. Optional: If you opted to use public API endpoints, you can select *Install into an existing VPC* to install your cluster into an existing VPC.
+
[NOTE]
====
If you opted to use private API endpoints, you must use an existing VPC and PrivateLink and the *Install into an existing VPC* and *Use a PrivateLink* options are automatically selected. With these options, the Red Hat Site Reliability Engineering (SRE) team can connect to the cluster to assist with support by using only AWS PrivateLink endpoints.
====
```